core/field_note.js

```diff
@@ -361,6 +361,11 @@ Blockly.FieldNote.prototype.dispose_ = function() {
     if (thisField.octaveUpMouseDownWrapper_) {
       Blockly.unbindEvent_(thisField.octaveUpMouseDownWrapper_);
     }
+    this.pianoSVG_ = null;
+    this.keySVGs_ = null;
```

Comment:
We tend to clear arrays by doing .length = 0. Do we need to do that here?
Differences I can see are that doing something like 
a1 = [1,2,3,4]
a2 = a1
a1 = null;

leaves a2 alone but  replacing a1=null with a1.length = 0 leaves a2 empty as well. Thoughts?